workers/generic-worker/schemas/multiuser_posix.yml

```diff
@@ -487,6 +487,31 @@ definitions:
       required:
       - taskId
       - artifact
+    - title: Indexed Content
+      description: |-
+        Content originating from a task artifact that has been indexed by the Taskcluster Index Service.
+
+        Since: generic-worker 50.2.0
+      type: object
+      properties:
+        namespace:
+          type: string
+          maxLength: 255
+        artifact:
+          type: string
+          maxLength: 1024
+        sha256:
+          type: string
+          title: SHA 256
+          description: |-
+            The required SHA 256 of the content body.
```

Comment:
The code suggests this may not be required (it's setting to `""` by default). Also, the schema doesn't list it as required. Probably needs to be reworded here and in the other schemas :)